syntax = "proto3";
message RequestMsg
{
    int32 cmdType = 1;
    string clientID = 2;
    string serverID = 3;
    string sign = 4;
    string data = 5;
}

message RespondMsg
{
    int32 status = 1;
    int32 seckeyID = 2;
    string clientID = 3;
    string serverID = 4;
    string data = 5;
}